### PR TITLE
Use Keycloak 19 instead of 16

### PR DESCRIPTION
**Pull request by fedinskiy**
### Summary

(Summarize the problem solved by this PR, and how to verify it manually)

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Dependency update
- [ ] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by pjgg** [comment]
Related to: https://github.com/quarkusio/quarkus/commit/517c971e56165a9888424fc9fbdcf858b2c37ac2

File: security/keycloak/src/test/java/io/quarkus/ts/security/keycloak/DevModeKeycloakDevServiceUserExperienceIT.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 @QuarkusScenario
 public class DevModeKeycloakDevServiceUserExperienceIT {
 
-    private static final String KEYCLOAK_VERSION = "18.0";
+    private static final String KEYCLOAK_VERSION = "19.0.1";
```
Comment (pjgg) [inline security/keycloak/src/test/java/io/quarkus/ts/security/keycloak/DevModeKeycloakDevServiceUserExperienceIT.java:18]: In this scenario we are forcing a different version than the default one, so could we use for example Keycloak 19.0.0 ?

**Review by pjgg — CHANGES_REQUESTED** [review]

File: security/keycloak/src/test/java/io/quarkus/ts/security/keycloak/DevModeKeycloakDevServiceUserExperienceIT.java (hunk 1 above)
Comment (fedinskiy) [reply security/keycloak/src/test/java/io/quarkus/ts/security/keycloak/DevModeKeycloakDevServiceUserExperienceIT.java]: Yes. Changed to 19.0.0

**Review by pjgg — APPROVED** [review]
LGTM

**Pull request merged**